x/concentrated-liquidity/types/pool_hooks.go

```diff
@@ -81,26 +99,6 @@ type AfterCreatePositionMsg struct {
 	UpperTick      int64             `json:"upper_tick"`
 }
 
-type BeforeAddToPositionMsg struct {
```

Comment:
Why did we decide to remove these?

Reply:
Please see the note here: https://github.com/osmosis-labs/osmosis/pull/6859/files#r1389677483